lxd/storage/drivers/generic_vfs.go

```diff
@@ -458,7 +458,14 @@ func genericVFSCreateVolumeFromMigration(d Driver, initVolume func(vol Volume) (
 		}
 
 		// Receive the block volume next (if needed).
-		if vol.IsVMBlock() || (IsContentBlock(vol.contentType) && vol.volType == VolumeTypeCustom) {
+		if volTargetArgs.ConversionFilePath != "" {
+			// If we are receiving block volume for conversion, store it into a backups directory,
```

Comment:
This comment isnt really in the right place - we don't know that the ConversionFilePath is pointing to the backups directory at this point.